justfile

```diff
@@ -39,12 +44,14 @@ generate-abi:
 	jq '.abi' out/ERC20.sol/ERC20.json > abi/ERC20.json
 	jq '.abi' out/IBCERC20.sol/IBCERC20.json > abi/IBCERC20.json
 	jq '.abi' out/IBCStore.sol/IBCStore.json > abi/IBCStore.json
-	abigen --abi abi/ICS02Client.json --pkg ics02client --type Contract --out e2e/interchaintestv8/types/ics02client/contract.go
```

Comment:
Just different ordering to have some level of separation between types generated for e2e tests only and the ones that should be public in `abigen/`

Reply:
That was done already